openshift-storage-libs/openshiftstoragelibs/openshift_ops.py

```diff
@@ -414,7 +417,8 @@ def _oc_create_app_dc_with_io_image(hostname, pvc_name, dc_name_prefix,
 
 def oc_create_app_dc_with_io(
         hostname, pvc_name, dc_name_prefix="autotests-dc-with-app-io",
-        replicas=1, space_to_use=1048576, label=None, image="cirros"):
+        args=None, replicas=1, space_to_use=1048576, label=None,
```

Comment:
`cmd_args` variable name is more clear to understand

Reply:
done